doc/platform_api/CMIS_and_C-CMIS_support_for_ZR.md

```diff
@@ -1586,3 +1586,104 @@ def write_cdb(port,cmd):
+    ---------------  ------  --------  --------  --------  -----------  -----------  ------------  -----------  -----------  -----------
+    Tx Power         dBm     -8.22     -8.23     -8.24     -5.0         -6.0         False         -16.99       -16.003      False
+    Rx Total Power   dBm     -10.61    -10.62    -10.62    2.0          0.0          False         -21.0        -18.0        False
+    Rx Signal Power  dBm     -40.0     0.0       40.0      13.0         10.0         True          -18.0        -15.0        True
+    CD-short link    ps/nm   0.0       0.0       0.0       1000.0       500.0        False         -1000.0      -500.0       False
+    PDL              dB      0.5       0.6       0.6       4.0          4.0          False         0.0          0.0          False
+    OSNR             dB      36.5      36.5      36.5      99.0         99.0         False         0.0          0.0          False
+    eSNR             dB      30.5      30.5      30.5      99.0         99.0         False         0.0          0.0          False
+    CFO              MHz     54.0      70.0      121.0     3800.0       3800.0       False         -3800.0      -3800.0      False
+    DGD              ps      5.37      5.56      5.81      7.0          7.0          False         0.0          0.0          False
+    SOPMD            ps^2    0.0       0.0       0.0       655.35       655.35       False         0.0          0.0          False
+    SOP ROC          krad/s  1.0       1.0       2.0       N/A          N/A          N/A           N/A          N/A          N/A
+    Pre-FEC BER      N/A     4.58E-04  4.66E-04  5.76E-04  1.25E-02     1.10E-02     0.0           0.0          0.0          0.0
+    Post-FEC BER     N/A     0.0       0.0       0.0       1000.0       1.0          False         0.0          0.0          False
+    EVM              %       100.0     100.0     100.0     N/A          N/A          N/A           N/A          N/A          N/A
+```
+#### 7.5 High Level Design
+
+##### 7.5.1 Configurations
+1. performance monitor enable - Global CLI to enable PM on all ports. 
+Before this configuration get implemented, PM will be enabled by default on all ports
+
+##### 7.5.2 Tables:
+1. The PM window table consists of the following:
+   - Window number
+   - Start timestamp of PM window
+   - Parameters from PM table
+
+2. Total 28 PM window slots per interface. 
+   - 15 slots of 60sec window (15mins of history).
+   - 12 slots of 15min window (3Hr of history) and 
+   - 1 slot of 24hr window.
+   
+
+ <img src="https://user-images.githubusercontent.com/97986478/217677958-fd532125-9483-4204-9a57-5317ec36b228.png" width="50%" height="50%">
+
+##### 7.5.3 Platform specific flags/inputs:
+"xcvrd_pm_poll_interval" - Platform to define the PM polling periodicity as 30sec or 60sec of the PM thread which will be fed as input argument. When the arg is not defined, default periodicity is 60sec. 
```

Comment:
if some platform defines the period as 30sec, what will be the behavior of commands "show int trans pm current 60sec" and "show int trans pm history 60sec"?  Where and how to change the "xcvrd_pm_poll_interval" if some vendor wants to change it?

Reply:
> if some platform defines the period as 30sec, what will be the behavior of commands "show int trans pm current 60sec" and "show int trans pm history 60sec"? Where and how to change the "xcvrd_pm_poll_interval" if some vendor wants to change it?

Only the granularity of updating the PM statistics will be changed. there will not be any behavior change in CLI command.
xcvrd_pm_poll_interval will be implemented the same way as skip_xcvrd_cmis_mgr in docker-pmon.supervisord.conf.j2.
By default the value is 60sec when the value is not provided by the platform.